test/exercism/html_formatter_test.rb

```diff
@@ -28,6 +29,16 @@ def test_python_docstring
     assert_equal 3, @doc.css('span.s').size, "There should be exactly 3 string tokens in the code"
     assert_equal '', @doc.css('span#L4').text.rstrip, "Line 4 should be blank"
     assert_equal '', @doc.css('span#L5').text.rstrip, "Line 5 should be blank"
+
+  end
+
+  def test_swift
+    load_sample 'swift', 'swift'
+
+    assert_equal 121, @doc.css('span[id^=L]').size, "Formatting should preserve linecount"
+    assert_equal "}", @doc.css('span#L121').text.rstrip, "Line 121 should have only a closing brace"
+    assert_equal "    }", @doc.css('span#L120').text.rstrip, "Line 120 should have a four space indent and closing brace"
```

Comment:
Don't use an instance variable `@doc` here (or anywhere,) when you can use a locally scoped return value from a method call. `doc = load_sample ...`